### DISCUSSION TITLE
Best method for adding particles?

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#357] [Answers & Questions] [answered]
Just curious to know if **PointCloudFromArray** is more performant or **MeshInstancer** for adding particles?

And what could be some use cases for spawning high number of particles through **MeshInstancer** if **PointCloudFromArray** already exists?

Thanks.

**Comment by pandrr** [comment, marked as answer]
pointcloud is for billboarded 2d sprites, mesh instancer is for 3d meshes

so complex shapes are slower than simple sprites

**Reply by monolithMktg** [reply]
Thank you very much.
